libs/dh/metering-point/feature-overview/src/lib/dh-customer-overview.component.ts

```diff
@@ -120,6 +122,8 @@ import { DhCanSeeValueDirective } from './dh-can-see-value.directive';
               </div>
             </ng-container>
           }
+        } @empty {
+          {{ null | dhEmDashFallback }}
```

Comment:
Praise: Forgot that for has en empty state :)

Reply:
Yes. It's really handy. :)